firmware/CMakeLists.txt

```diff
@@ -30,33 +30,46 @@ include(${CMAKE_CURRENT_LIST_DIR}/mbed/arm_mbed.cmake)
 # enable C++ exceptions
 set(MBED_CMAKE_CXX_FLAGS "${MBED_CMAKE_CXX_FLAGS} -fexceptions")
 # enable C++11
-set(MBED_CMAKE_CXX_FLAGS "${MBED_CMAKE_CXX_FLAGS} -std=c++11")
+# set(MBED_CMAKE_CXX_FLAGS "${MBED_CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
wait...is setting both any different from only setting the highest cpp std version?